client/.eslintrc.js

```diff
@@ -33,8 +33,15 @@ module.exports = {
     'require-jsdoc': 'off',
     'valid-jsdoc': 'off',
     'react/display-name': 'off',
-    'no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
-    '@typescript-eslint/no-unused-vars': 'off',
+    'no-unused-vars': 'off',
+    '@typescript-eslint/no-unused-vars': [
+      'error',
+      {
+        argsIgnorePattern: '^_',
+        varsIgnorePattern: '^_',
```

Comment:
Just wondering why we'd want to ignore unused vars?

Reply:
if we want like the second var but no the first

Reply:
That's unused args, which we already have. Was wondering about the `vars`, specifically.

Reply:
ohh true `varsIgnorePattern` 🏃‍♂️

Reply:
yes right no real reason to have it in